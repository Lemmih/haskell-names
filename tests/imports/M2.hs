module M2 where

import Prelude (DataType(..), function)
