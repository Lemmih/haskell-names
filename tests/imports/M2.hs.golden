Table (
  fromList
    [ ( GName
          ""
          "Constructor1"
      , fromList
          [ SymConstructor
              { sv_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "Constructor1"
                                }
              , sv_fixity = Nothing
              , sv_typeName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "DataType"
                                }
              }
          ]
      )
    , ( GName
          ""
          "Constructor2"
      , fromList
          [ SymConstructor
              { sv_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "Constructor2"
                                }
              , sv_fixity = Nothing
              , sv_typeName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "DataType"
                                }
              }
          ]
      )
    , ( GName
          ""
          "function"
      , fromList
          [ SymValue
              { sv_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "function"
                                }
              , sv_fixity = Nothing
              }
          ]
      )
    , ( GName
          "Prelude"
          "Constructor1"
      , fromList
          [ SymConstructor
              { sv_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "Constructor1"
                                }
              , sv_fixity = Nothing
              , sv_typeName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "DataType"
                                }
              }
          ]
      )
    , ( GName
          "Prelude"
          "Constructor2"
      , fromList
          [ SymConstructor
              { sv_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "Constructor2"
                                }
              , sv_fixity = Nothing
              , sv_typeName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "DataType"
                                }
              }
          ]
      )
    , ( GName
          "Prelude"
          "function"
      , fromList
          [ SymValue
              { sv_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "function"
                                }
              , sv_fixity = Nothing
              }
          ]
      )
    ] ) (
  fromList
    [ ( GName
          ""
          "DataType"
      , fromList
          [ SymData
              { st_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "DataType"
                                }
              , st_fixity = Nothing
              }
          ]
      )
    , ( GName
          "Prelude"
          "DataType"
      , fromList
          [ SymData
              { st_origName = OrigName
                                { origPackage = Nothing
                                , origGName = GName
                                                "Prelude"
                                                "DataType"
                                }
              , st_fixity = Nothing
              }
          ]
      )
    ] )